interpreter/k8s/src/main/scala/io/buoyant/transformer/k8s/DaemonSetTransformerInitializer.scala

```diff
@@ -16,6 +16,8 @@ class DaemonSetTransformerInitializer extends TransformerInitializer {
   override val configId = "io.l5d.k8s.daemonset"
 }
 
+object DaemonSetTransformerInitializer extends DaemonSetTransformerInitializer
+
```

Comment:
not sure its worth it having a different object just so instantiation is easier

Reply:
w/o having this different object the test file DaemonSetTransformerTest won't compile

Reply:
Why.... ?

Reply:
Parser.objectMapper(yaml, Iterable(Seq(DaemonSetTransformerInitializer))) -> Parser.objectMapper(yaml, Iterable(Seq(new DaemonSetTransformerInitializer))), would that work ?